recipes/tsil/all/CMakeLists.txt

```diff
@@ -0,0 +1,83 @@
+cmake_minimum_required(VERSION 3.4)
+project(TSIL C)
+
+include(conanbuildinfo.cmake)
+conan_basic_setup()
+
+if(NOT CMAKE_BUILD_TYPE)
+  set(CMAKE_BUILD_TYPE Release)
+endif()
+
+if(NOT TSIL_SIZE)
+  set(TSIL_SIZE "TSIL_SIZE_LONG") # default
+endif()
+
+if(TSIL_SIZE STREQUAL "TSIL_SIZE_DOUBLE")
+  add_definitions(-DTSIL_SIZE_DOUBLE)
+elseif(TSIL_SIZE STREQUAL "TSIL_SIZE_LONG")
+  add_definitions(-DTSIL_SIZE_LONG)
```

Comment:
prefer `target_compile_definitions()`

Reply:
Thanks for the suggestion! I've changed it in commit 53ca6fe8